src/geosPythonInterface/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+
+set( python_headers geosPythonInterface.hpp )
+
+set( python_sources geosPythonInterface.cpp )
+
+set( dependencyList ${extraComponentsLinkList} ${externalComponentsLinkList} Python3::Python )
+
+blt_add_library( NAME geosPythonInterface
+                 SOURCES ${python_sources}
+                 HEADERS ${python_headers}
+                 DEPENDS_ON ${dependencyList}
+                 SHARED TRUE
+                 CLEAR_PREFIX TRUE
+               )
+
+add_custom_command( TARGET geosPythonInterface
+                    PRE_BUILD
+                    COMMAND ${Python3_EXECUTABLE} -m venv ${CMAKE_BINARY_DIR}/lib/PYGEOS --system-site-packages )
```

Comment:
We should also refactor `lib/PYGEOS` to something like `lib/GEOS_PYTHON` to avoid any potential import errors.